pkg/serviceaccounts/client.go

```diff
@@ -1,119 +1,11 @@
 package serviceaccounts
 
 import (
-	"fmt"
-	"time"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	restclient "k8s.io/client-go/rest"
 	kapi "k8s.io/kubernetes/pkg/api"
-	kclientsetexternal "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
-	kclientsetinternal "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
-
-	"github.com/openshift/origin/pkg/client"
```

Comment:
@stevekuznetsov this transitive wasn't picked up before.  Weird.  Known?

Reply:
Did we merge upstream fix and rebase to grab it? There was an issue with prefixing.

Reply:
> Did we merge upstream fix and rebase to grab it? There was an issue with prefixing.

Ok, known.  Thanks.